src/tree-view.ts

```diff
@@ -44,9 +44,13 @@ export class CodebaseProvider
 				names: name,
 			});
 
+			const mapDefinitions = mapDefinitionsByName(name);
+			const termDefinitions = mapDefinitions(definition.termDefinitions);
+			const typeDefinitions = mapDefinitions(definition.typeDefinitions);
+
 			const signatureTokens =
-				definition.termDefinitions[name]?.signature ??
-				definition.typeDefinitions[name]?.typeDefinition.contents;
+				termDefinitions[name]?.signature ??
+				typeDefinitions[name]?.typeDefinition.contents;
```

Comment:
this is unrelated to my functionality actually, and serves to address the bug i discuss in https://github.com/tom-sherman/unison-vscode-ui/pull/20/files#r1402817184